package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-doc-statement-publisher",
-  "version": "2.1.22",
+  "version": "2.1.24",
```

Comment:
Shouldn't be a double version jump, however I'd also argue this is a sufficiently big addition to justify v2.2.0

Reply:
I think I agree with the jump to 2.2.0 also.